bin/swagger-jsdoc.js

```diff
@@ -118,11 +118,12 @@ fs.readFile(program.definition, 'utf-8', function(err, data) {
   }
 
   // If there's no argument passed, but the user has defined Apis in
-  // the definition file, pass them them onwards.
+  // the definition file, pass them them onwards and remove from definitions
   if (program.args.length === 0 &&
     swaggerDefinition.apis &&
     swaggerDefinition.apis instanceof Array) {
     program.args = swaggerDefinition.apis;
+    delete swaggerDefinition.apis;
```

Comment:
Hi @JesseObrien as I'm not a big fan of mutating things, I'd suggest a bit different approach which will basically make the changes also a bit more readable.
1) Create a new variable `swaggerFiles` before https://github.com/Surnet/swagger-jsdoc/blob/master/bin/swagger-jsdoc.js#L19 For readability mainly
2) By default it contains program.args
3) In your check override this only when necessary (with swaggerDefinition.apis)
4) Use it in `createSpecification(swaggerDefinition, swaggerFiles, output);`

Again, it's mainly for avoiding removing things that (who knows) might be necessary later in the program when it's extended in future. It's also making the changes more readable.